lib/tasks/test_vmdb.rake

```diff
@@ -8,56 +8,40 @@ namespace :test do
       # in case we are called from an engine or plugin, the task might be namespaced under 'app:'
       # i.e. it's 'app:test:vmdb:setup'. Then we have to call the tasks in here under the 'app:' namespace too
       app_prefix = rake_task.name.chomp('test:vmdb:setup')
-      if ENV['PARALLEL']
-        database_config = Pathname.new(__dir__).expand_path + "../../config/database.yml"
-        if File.readlines(database_config).grep(/\A\s+database:.+TEST_ENV_NUMBER/).size > 0
-          require 'parallel_tests'
-          ParallelTests::CLI.new.run(["--type", "rspec"] + ["-e", "bin/rake #{app_prefix}test:setup_db"])
-        else
-          puts "Oops! Your database.yml doesn't appear to support parallel tests!"
-          puts "Update your config/database.yml with TEST_ENV_NUMBER as seen in the example (database.pg.yml), then try again."
-          exit(1)
-        end
-      else
-        Rake::Task["#{app_prefix}test:setup_db"].invoke
-      end
+      Rake::Task["#{app_prefix}test:setup_db"].invoke
     end
   end
 
-  desc "Run all vmdb specs; Use PARALLEL=true to run in parallel."
+  desc "Run all vmdb specs"
   task :vmdb do
-    if ENV['PARALLEL']
-      Rake::Task['test:vmdb_parallel'].invoke
-    else
-      Rake::Task['test:vmdb_sequential'].invoke
-    end
+    Rake::Task['test:vmdb'].invoke
```

Comment:
test:vmdb doesn't exist?  It's a namespace on line 5 above ☝️

Reply:
This is a circular reference.  This task _is_ test:vmdb